compat/mangle.json

```diff
@@ -23,7 +23,6 @@
       "$_render": "__r",
       "$_hook": "__h",
       "$_catchError": "__e",
-      "$_catchRender": "__E",
```

Comment:
I'm not sure if we can remove the mangle setting here as `Suspense` in compat still relies on a consistent name with core.